Mathlib/Tactic/Simp/DivisorsAntidiag.lean

```diff
@@ -0,0 +1,50 @@
+/-
+Copyright (c) 2025 Yaël Dillies, Paul Lezeau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Paul Lezeau
+-/
+import Mathlib.Data.Finset.Sort
+import Mathlib.Lean.Expr.Lit
+import Mathlib.NumberTheory.Divisors
+import Mathlib.Tactic.NormNum
+
+/-!
+# Simproc for `Int.divisorsAntidiag`
+-/
+
+open
+  Qq
+  Lean
+    Meta
+      Simp
+    Elab
+      Term
+    Parser
+      Tactic
```

Comment:
I think this is a little unhinged, especially because the structure this implies does not reflect the actual behavior (this opens `Lean.Elab.Tactic` too)

Reply:
I am aware (and sad about it), but anyway this is still WIP and is currently here only for me to understand what's going on